### PR TITLE
Improve Ui and Fix bug

### PR DESCRIPTION
**Pull request by cuclt**

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class Category < ApplicationRecord
   has_many :lessons
   has_many :questions
 
-  validates :title, presence: true, length: {maximum: 100}
+  validates :title, presence: true, length: {maximum: 100}, uniqueness: {case_sensitive: false}
```
Comment (nvtanh) [inline app/models/category.rb]: quá 80 kí tự

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
+        <div class="form-group">
+          <%= f.label t("email") %>
+          <%= f.email_field :email, class: "form-control" %>
+        </div>
+        <div class="form-group">
+          <%= f.label t("password") %>
+          <%= f.password_field :password, class: "form-control" %>
+        </div>
+        <div class="form-group">
+          <%= f.label t("password_confirmation"), "Confirmation" %>
```
Comment (nvtanh) [inline app/views/users/new.html.erb]: I18n

File: app/views/users/edit.html.erb [hunk 3]
```diff
@@ -1,16 +1,30 @@
-<div class="row">
-  <div class="col-md-6 col-md-offset-3">
-    <%= form_for @user do |f| %>
+<div class="col-md-12">
+  <div class="box">
+    <div class="col-md-6 col-sm-offset-3">
+      <h1 class="text-center"><%= t "edit_profile" %></h1>
+      <hr>
+      <%= form_for @user do |f| %>
       <%= render "shared/error_messages", object: f.object %>
```
Comment (dieunb) [inline app/views/users/edit.html.erb]: indent

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -1,23 +1,30 @@
...
-
-      <%= f.submit t("create_account"), class: "btn btn-primary" %>
-    <% end %>
+<div class="col-md-12 box">
+  <div class="box">
+    <div class="col-md-6 col-sm-offset-3">
+      <h1 class="text-center"><%= t "signup" %></h1>
+      <hr>
+      <%= form_for @user do |f| %>
+      <%= render "shared/error_messages", object: f.object %>
```
Comment (dieunb) [inline app/views/users/new.html.erb]: indent

**Pull request merged**